src/usuarios/usuarios-controlador.js

```diff
@@ -86,20 +88,44 @@ async function verificaRefreshToken(token) {
   return payload;
 }
 
+async function enviaEmailConfirmacao(email) {
+  // faz o curso com a conta teste + 'para saber mais' com dicas para produção?
+
+  const contaTeste = await nodemailer.createTestAccount();
+
+  const transportador = nodemailer.createTransport({
+    host: 'smtp.ethereal.email',
+    auth: contaTeste
+  });
+
+  const info = await transportador.sendMail({
+    from: 'noreply@blogdocodigo.com.br',
+    to: email,
+    subject: 'Confirmação de e-mail',
+    text: 'confirme aqui!!'
+  });
+
+  console.log('URL:' + nodemailer.getTestMessageUrl(info));
+}
+
 module.exports = {
   async adiciona(req, res) {
     const { nome, email, senha } = req.body;
 
     try {
       const usuario = new Usuario({
         nome,
-        email
+        email,
+        emailConfirmado: false
       });
 
       await usuario.adicionaSenha(senha);
 
       await usuario.adiciona();
 
+      // sem await a request não trava mas não sei como tratar eventuais erros
+      enviaEmailConfirmacao(email);
```

Comment:
Aqui vc pode ler mais: https://itnext.io/error-handling-with-async-await-in-js-26c3f20bc06a

 Mas o resumo é que vc pode encadear a promisse com o .catch ( da mesma maneira que fazemos com  o .then mas o catch só pega erros)

Reply:
Ah, isso sim! Em algum commit depois desse eu uso o catch pra imprimir os erros no terminal. Minha pergunta era mais se tem algo além disso que dá pra fazer. No máximo, daria pra tentar o envio de novo caso dê problema, né?